### PR TITLE
#U44 refactor: blog section 

### PR DESCRIPTION
**Pull request by chrisostome01**
# PR Description

This pull request aims to refactor the blog section 

# Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you  Make sure you check every completed task. list them in this style

- [x] Removes unnecessary codes

# Description of tasks that were expected to be completed and that are not yet completed

- [x] none

# How this may be tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependencies 
2. run `git checkout refact/blog-section`
3. run `npm run dev`
4. navigate to localhost:3000 
5. here you can jump to the blog section and scroll right or left  

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: components/sections/blogs/Posts.jsx [hunk 1]
```diff
@@ -1,17 +1,17 @@
 /* eslint-disable react-hooks/exhaustive-deps */
 import { useEffect, useRef, useState } from "react";
-import Blog from "../ui/Blog";
-import Layout from "../layout/Layout";
+import Layout from "../../layout/Layout";
 import blogData from "/utils/blogData";
-import RightArrow from "../vectors/Arrow";
+import RightArrow from "../../vectors/Arrow";
+import RenderPosts from "./RanderPosts"
```
Comment (serapieTuyishime) [inline components/sections/blogs/Posts.jsx]: Can you fix the **RanderPosts**, there is a typo. it should be `RenderPost`

**Review by serapieTuyishime — CHANGES_REQUESTED** [review]

**Review by Yvad60 — APPROVED** [review]
Nice work @sezeranoJchrisostome. Consider Serapie's comment though otherwise this looks good to me

File: components/sections/blogs/Posts.jsx (hunk 1 above)
Comment (chrisostome01) [reply components/sections/blogs/Posts.jsx]: Nice

**Review by serapieTuyishime — APPROVED** [review]
LGTM

File: components/sections/blogs/Posts.jsx [hunk 2]
```diff
@@ -86,26 +76,7 @@ export default function BlogPost() {
...
-                  key={index}
-                  className="bg-muted-50 rounded-2.3xl px-2.5 py-01"
-                >
-                  <p className="text-lg font-normal text-muted-200">{value}</p>
-                </div>
-              ))}
-            </Blog>
-          );
-        })}
+        <RanderPost blogs={blogs} />
```
Comment (Yvad60) [inline components/sections/blogs/Posts.jsx:79]: I thought this would be `RenderPost`

**Review by Yvad60 — APPROVED** [review]
Approved

**Pull request merged**